content/en/methods/notifications_alpha.md

```diff
@@ -0,0 +1,598 @@
+description: Receive notifications for activity on your account or statuses.
+---
+
+<style>
+#TableOfContents ul ul ul {display: none}
+</style>
+
+{{< hint style="warning" >}}
+This page documents API endpoints that are not finalized. We welcome feedback on them and you are free to implement them, but we recommend not having these implementations in a release, as they are likely to change and we will not ensure backward compatibility if they do.
+{{</ hint >}}
+
+This page is about grouped notifications, which we implemented server-side so that:
+- grouping is consistent across clients
+- clients do not run into the issue of going through entire pages that do not contribute to any new group; instead, notifications are already deduplicated server-side
+
+The API shape is a bit different from the non-grouped notifications, because large notification groups usually tend to involve the same accounts, and moving accounts to a root key can avoid a lot of duplication, resulting in less server-side work and smaller network payloads.
+
+## Get all grouped notifications {#get-grouped}
+
+```http
+GET /api/v2_alpha/notifications HTTP/1.1
+```
+
+Return grouped notifications concerning the user. This API returns Link headers containing links to the next/previous page. However, the links can also be constructed dynamically using query params and `id` values.
+
+Notifications of type `favourite` or `reblog` with the same type and the same target made in a similar timeframe are given a same `group_key` by the server, and querying this endpoint will return aggregated notifications, with only one object per `group_key`. Other notification types (such as `follow`) may be grouped in the future.
+
+Types to filter include:
+- `mention` = Someone mentioned you in their status
+- `status` = Someone you enabled notifications for has posted a status
+- `reblog` = Someone boosted one of your statuses
+- `follow` = Someone followed you
+- `follow_request` = Someone requested to follow you
+- `favourite` = Someone favourited one of your statuses
+- `poll` = A poll you have voted in or created has ended
+- `update` = A status you boosted with has been edited
+- `admin.sign_up` = Someone signed up (optionally sent to admins)
+- `admin.report` = A new report has been filed
+
+**Returns:** [GroupedNotificationsResults](#GroupedNotificationsResults)\
+**OAuth:** User token + `read:notifications`\
+**Version history:**\
+4.3.0 - added
+
+#### Request
+
+##### Headers
+
+Authorization
+: {{<required>}} Provide this header with `Bearer <user token>` to gain authorized access to this API method.
+
+##### Query parameters
+
+max_id
+: String. All results returned will be lesser than this ID. In effect, sets an upper bound on results.
+
+since_id
+: String. All results returned will be greater than this ID. In effect, sets a lower bound on results.
+
+min_id
+: String. Returns results immediately newer than this ID. In effect, sets a cursor at this ID and paginates forward.
```

Comment:
What is `ID` in this context? Group keys?

Reply:
No, they are notification IDs! I will make that explicit.

Reply:
Updated, hopefully this is clearer now!

Reply:
Yes!